src/components/SlateEditor/plugins/link/EditLink.jsx

```diff
@@ -42,6 +40,30 @@ const createLinkData = (href, targetRel) => ({
   },
 });
 
+const isNDLAArticleUrl = url =>
+  /^https:\/\/((.*)\.)?ndla.no\/article\/\d*/.test(url);
+const isNDLASubjectUrl = url =>
+  /^https:\/\/((.*)\.)?ndla.no\/subjects\/(.*)/.test(url);
+
+const isPlainId = url => /^\d+/.test(url);
+
+const getArticleIdFromUrl = async href => {
```

Comment:
En ressurs kan være en læringssti. Bør ta høgde for det.

Reply:
Enig!
Tar å setter igang med backenden nå :)